src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.data.person;
+
+
+/**
+ * Represents a Person's block  in the address book.
```

Comment:
may be needs a small rewording.
Block belongs to the address; not necessarily address book, no?